src/Microsoft.ML.LightGbm/LightGbmMulticlassTrainer.cs

```diff
@@ -109,8 +111,7 @@ static Options()
                 var res = base.ToDictionary(host);
 
                 res[GetOptionName(nameof(Sigmoid))] = Sigmoid;
-                if(EvaluationMetric != EvaluateMetricType.Default)
-                    res[GetOptionName(nameof(EvaluateMetricType))] = GetOptionName(EvaluationMetric.ToString());
+                res[GetOptionName(nameof(EvaluateMetricType))] = GetOptionName(EvaluationMetric.ToString());
```

Comment:
>EvaluateMetricType [](start = 41, length = 18)

wouldn't it be nice to have test?

Reply:
You mean like check if EvaluationMetric exists here?

Reply:
[https://github.com/dotnet/machinelearning/issues/3761](https://github.com/dotnet/machinelearning/issues/3761) that issue showing certain code failure.
You PR claims it fixes it.
Would it make sense to add that code from issue to test we don't fail anymore?

---
In reply to: [290089536](https://github.com/dotnet/machinelearning/pull/3815#discussion_r290089536) [](ancestors = 290089536)